deptrack-devops/src/Devops/Graph.hs

```diff
@@ -148,6 +152,46 @@ type Broadcast = (OpUniqueId,CheckResult,Stability,WantedDirection) -> IO ()
 noBroadcast :: Broadcast
 noBroadcast = const (return ())
 
+-- | Turn-up a graph sequentially
+-- Topologically sort all nodes of the graph then run corresponding operations
```

Comment:
Please stress the fact that topSort will be deterministic in the comment because that's the one big benefit over just taking a Semaphore in `traverseOpGraph`.